packages/terra-collapsible-menu-view/src/CollapsibleMenuView.jsx

```diff
@@ -45,6 +42,24 @@ const propTypes = {
    * Aligns the menu to the start of the container
    */
   isStartAligned: PropTypes.bool,
+
+  /**
+   * @private
```

Comment:
Should this be marked a private?  isn't it used externally in your tests?

Reply:
Yes since it's not intended to be used by consumers. I still would like to test that it works though, I'm not sure what the standard is but can you not have private props that are still tested?

Reply:
If it's only for internal use then also using them for tests are fine imo.